doc/man7/provider-signature.pod

```diff
@@ -518,8 +518,12 @@ A getter that returns 1 if a signature verification operation acted on
 a raw message, or 0 if it verified a predigested message.  A value of 0
 indicates likely non-approved usage of the FIPS provider.  This flag is
 set when any signature verification initialisation function is called.
-It is also set to 1 when any signing operation is performed to signify
-compliance.  See FIPS 140-3 IG 2.4.B for further information.
+For RSA signing operations it is also set to 1 when acting on a raw
```

Comment:
I dont think we should be reusing "verify-message" for this...

Reply:
Well, it was used in both signing and verification since inception.

If we want to keep it descriptive in the natural language, "sign-message" would be a better term when signing, and not raise "verify-message" during signing anymore.